src/lib/db/server.js

```diff
@@ -51,6 +51,10 @@ export function createServer(serverConfig) {
         return server.db[dbName];
       }
 
+      if(dbName === null && server.config.client === 'postgresql') {
```

Comment:
I think you need `serverConfig.client`?

Reply:
Both possible, i changed it to `serverConfig` now.